example/package.mill

```diff
@@ -65,7 +65,10 @@ object `package` extends RootModule with Module {
     object basic extends Cross[ExampleCrossModule](build.listIn(millSourcePath / "basic"))
   }
   object pythonlib extends Module {
-    object basic extends Cross[ExampleCrossModule](build.listIn(millSourcePath / "basic"))
+    //TODO: commented-out to prevent doc-generation from failing. Re-enable this.
+    // object basic extends Cross[ExampleCrossModule](build.listIn(millSourcePath / "basic"))
+
+    object dependencies extends Cross[ExampleCrossModule](build.listIn(millSourcePath / "dependencies"))
```

Comment:
If we're going to use `ExampleCrossModule`, this doesn't inherit from any upstream examples, so we can leave out all the `////SNIPPET` things